### PR TITLE
Removed oss-library-gradle-plugin with maven-publish

### PR DESCRIPTION
**Pull request by tanya732**
Changes

- Modified Github workflow and gradle plugins

### References

Please include relevant links supporting this change such as a:

- https://github.com/auth0/Auth0.Android/pull/739

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,44 +1,37 @@
...
+}
+
+
 plugins {
     id 'java'
     id 'jacoco'
-    id 'com.auth0.gradle.oss-library.java'
+    id 'me.champeau.gradle.japicmp' version '0.2.9'
+//    id 'java-gradle-plugin'
+//    id "com.gradle.plugin-publish" version "0.12.0"
```
Comment (jimmyjames) [inline build.gradle:19]: any reason to keep these commented-out lines?

File: build.gradle [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
 }
 
 repositories {
+//    jcenter()
+//    google()
+//    gradlePluginPortal()
```
Comment (jimmyjames) [inline build.gradle:25]: can we remove these commented lines?

**Review by jimmyjames — COMMENTED** [review]
❤️ I love getting rid of the plugin, or at least the publishing pieces of it! My only question would be that since we have many java SDKs, it seems like there will be a lot of common build script code replicated across the libraries. Just wondering if there's a way to reuse that code, e.g., maybe in an updated plugin that only has that shared code. But I will let you and the team make that decision 👍

**Comment by tanya732** [comment]
This PR is not being used. Refer - https://github.com/auth0/auth0-java/pull/665

**Pull request closed without merging**